src/util/guard_bdd.h

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************\
+
+Module: Guard Data Structure
+
+Author: Romain Brenguier, romain.brenguier@diffblue.com
+
+\*******************************************************************/
+
+/// \file
+/// Guard Data Structure
+/// Implementation using BDDs
+
+#ifndef CPROVER_UTIL_GUARD_BDD_H
+#define CPROVER_UTIL_GUARD_BDD_H
+
+#include <iosfwd>
+#include <memory>
+
+#include "bdd.h"
+#include "make_unique.h"
+#include "std_expr.h"
+
+struct guard_bdd_managert
```

Comment:
❓ 🚫  Who owns this manager object?

Reply:
There are several ones: in goto_checkt, goto_symext, rd_range_domaint, dep_graph_domaint, and goto_instrument_parse_options